File: WebsiteTinTuc.User/WebsiteTinTuc.User/Models/Post.cs
﻿using System;

namespace WebsiteTinTuc.User.Models
{
    public class Post
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public string PostUrl { get; set; }
        public string AgencyName { get; set; }
        public bool IsCreate { get; set; }
    }
}

File: WebsiteTinTuc.User/WebsiteTinTuc.User/VueApp/tin-tuc-app/store/interfaces/post.ts
import { Guid } from "guid-typescript";
import Entity from "./entity";

export default interface Post extends Entity<Guid>, PostTopProminent {
    content: string;
}

export interface PostTopProminent extends Entity<Guid> {
    companyName: string;
    title: string;
    postUrl: string;
}